src/components/AppHeader/SkeletonLoader/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ContentLoader from 'react-content-loader';
+
+type AccountsInfoLoaderProps = {
+    is_logged_in: boolean;
+    is_mobile: boolean;
+    speed: number;
```

Comment:
```suggestion
    ```suggestion
    isLoggedIn: boolean;
    isMobile: boolean;
    speed: number
```